### PR TITLE
Mock BIE Kafka: add SSL authentication

### PR DESCRIPTION
**Pull request by yoomlam**
## What was the problem?

Mock BIE Kafka was dissimilar to the actual BIE Kafka b/c it did not require SSL authentication.

Associated tickets or Slack threads:
- #1679
- #1823

## How does this fix it?

* Mock BIE Kafka: add SSL authentication
* Update tests

### CODE REVIEW

**Comment by yoomlam** [comment]
[3. Continuous Integration (run tests and check](https://github.com/department-of-veterans-affairs/abd-vro/actions/runs/5798169967) passed

File: svc-bie-kafka/src/main/resources/application-integration-test.yaml
Comment (yoomlam) [inline svc-bie-kafka/src/main/resources/application-integration-test.yaml:1]: for `BieKafkaApplicationTest`

**Comment by yoomlam** [comment]
[3. Continuous Integration (run tests and check](https://github.com/department-of-veterans-affairs/abd-vro/actions/runs/5799044619) passed

**Review by chengjie8 — APPROVED** [review]
The ssl settings in yaml files is slightly different (with dashes), I think spring would take either version. 
LGTM!

**Pull request merged**